src/Orleans.Core.Legacy/Configuration/NodeConfiguration.cs

```diff
@@ -204,22 +204,22 @@ public NodeConfiguration()
             this.AddressType = AddressFamily.InterNetwork;
             this.ProxyGatewayEndpoint = null;
 
-            this.MaxActiveThreads = SchedulingOptions.DEFAULT_MAX_ACTIVE_THREADS;
-            this.DelayWarningThreshold = SchedulingOptions.DEFAULT_DELAY_WARNING_THRESHOLD;
-            this.ActivationSchedulingQuantum = SchedulingOptions.DEFAULT_ACTIVATION_SCHEDULING_QUANTUM;
-            this.TurnWarningLengthThreshold = SchedulingOptions.DEFAULT_TURN_WARNING_THRESHOLD;
-            this.EnableWorkerThreadInjection = SchedulingOptions.DEFAULT_ENABLE_WORKER_THREAD_INJECTION;
+            this.MaxActiveThreads = Math.Max(4, Environment.ProcessorCount);
+            this.DelayWarningThreshold = TimeSpan.FromMilliseconds(10000);
+            this.ActivationSchedulingQuantum = TimeSpan.FromMilliseconds(100);
+            this.TurnWarningLengthThreshold = TimeSpan.FromMilliseconds(200);
```

Comment:
generally speaking we don't use random numbers. Can you add those numbers as DEFAULXXX in this file so the code will be more readable?